chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
>My argument for `var` has been that being able to (re)initialize variables at function scope close to where they're used without worrying about whether they're declared above or below is a good feature.

I don't think I've ever seen a case where that is a good idea. If you're declaring a variable inside a block that is intended to be visible outside of block, you should be making it visible explicitly to help readability. If you are redeclaring a variable in the same block, you are clearly not aware of the other declaration, or you wouldn't be doing so, so being informed of that is a great idea.

The issue with declaring everything up top for me is that half way down the function, when a new variable starts being used, I have to remember if it was declared at top. Most of the time, I just assume it was.